sandbox-tools/merge-repos/src/mergeStagingToMain.ts

```diff
@@ -348,7 +348,7 @@ async function getStagingRepo(git: SimpleGit, repoName: string, details: IRepoDe
 
         let commitMessage = stagingDetails.commitDetails.message;
 
-        commitMessage += `\n### Moving ${fileDetails.srcPath} to ${fileDetails.destPath}`;
+        //commitMessage += `\n### Moving ${fileDetails.srcPath} to ${fileDetails.destPath}`;
```

Comment:
This commit was just adding noise to the commit message (always moving the files to the root)